### DISCUSSION TITLE
Language settings

### DISCUSSION THREAD

**Discussion by kozaka-tv** [night/betterttv#6422] [Ideas]
It is nice, if you read from my browser to know what language is set there, but I would like to have an option to switch to a different supported language on the web page of BTTV.

At least, I could not find any opportunity to switch to English from Hungarian if I wish.

**Comment by kozaka-tv** [comment]
Bump

**Reply by night** [reply]
This is supported already by nature of Twitch/YouTube language settings. If you change the language of Twitch or YouTube, we will respect that language. If we're unable to detect that language, we default to the browser's language.

**Reply by kozaka-tv** [reply]
> This is supported already by nature of Twitch/YouTube language settings. If you change the language of Twitch or YouTube, we will respect that language. If we're unable to detect that language, we default to the browser's language.

I am speaking about the option to change language on the https://betterttv.com/ page. What is now not supported. I do not want to change anything in my browser only because of a webpage. The point is, NOT to use any of the (from you supported) browser default language, rather than be able to choose an another one.

Hungarian is supported, working well. But if I want to read in English, I can not do this, without change the complete language of my browser.
